aiocache/decorators.py

```diff
@@ -58,6 +62,7 @@ def __init__(
         ttl=SENTINEL,
         namespace=None,
         key_builder=None,
+        value_checker=None,
```

Comment:
```suggestion
        skip_cache_func=lambda r: False,
```

How about we just put a default function and not need to test for `None`?

Reply:
Its an antipattern
https://www.flake8rules.com/rules/E731.html

Reply:
The function returns True, it's never going to appear in a traceback..

Reply:
Also, the antipattern is assigning to a variable. lambdas exist to be passed directly into a function.
`foo(lambda: ...)` is allowed
`f = lambda: ...` is not

Reply:
So, my understanding is that when you do `f = lambda: ...`, you are essentially trying to give a name to a lambda (`f` in this example). But, when printing it out etc. you won't see that name. That is an antipattern.

In this case, we are not trying to assign a name to the lambda, it is the default value which would not have a name associated with it.